File: content/43/a.php
<?php
$s = 'June';
# example 1
$n = strpos($s, 'Ju');
var_dump($n === 0);
# example 2
$n = strpos($s, 'un');
var_dump($n !== false);
# example 3
$n = stripos($s, 'ju');
var_dump($n === 0);
# example 4
$s2 = $s[0];
var_dump($s2 == 'J');
# example 5
$s2 = $s[-1];
var_dump($s2 == 'e');

File: content/43/a.go
package main
import "strings"

func main() {
   s := "June"
   // example 1
   b := strings.HasPrefix(s, "Ju")
   // example 2
   b2 := strings.Contains(s, "un")
   // example 3
   b3 := strings.HasSuffix(s, "ne")
   // print
   println(b, b2, b3)
}
